### PR TITLE
feat(graphql): add tags to graphql spans

### PR DESCRIPTION
**Pull request by mabdinur**
## Description

The tags listed below are set on in the node and .net tracers. These tags should also be here for feature parity:
 
### `graphql.execute` span
 - graphql.operation.name
 - graphql.operation.type
 - graphql.source

### `graphql.validate` span
 - graphql.source

### `graphql.parse` span
 - graphql.source
 
## Checklist
- [x] Title must conform to [conventional commit](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional).
- [x] Add additional sections for `feat` and `fix` pull requests.
- [x] Ensure tests are passing for affected code.
- [x] [Library documentation](https://github.com/DataDog/dd-trace-py/tree/1.x/docs) and/or [Datadog's documentation site](https://github.com/DataDog/documentation/) is updated. Link to doc PR in description.

## Reviewer Checklist
- [x] Title is accurate.
- [x] Description motivates each change.
- [x] No unnecessary changes were introduced in this PR.
- [x] PR cannot be broken up into smaller PRs.
- [x] Avoid breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
- [x] Tests provided or description of manual testing performed is included in the code or PR.
- [x] Release note has been added for fixes and features, or else `changelog/no-changelog` label added.
- [ ] All relevant GitHub issues are correctly linked.
- [ ] Backports are identified and tagged with Mergifyio.
- [ ] Add to milestone.

### CODE REVIEW

File: releasenotes/notes/graphql-add-source-operation-tags-64e51c371fd25a1e.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (Kyle-Verhoog) [inline releasenotes/notes/graphql-add-source-operation-tags-64e51c371fd25a1e.yaml:1]: probably don't need this as graphql hasn't been released yet

**Review by Kyle-Verhoog — COMMENTED** [review]
lgtm to me besides the note

**Review by Kyle-Verhoog — APPROVED** [review]
👍 as long as these are roughly standardized across the other libraries i'm good with this

**Review by ZStriker19 — APPROVED** [review]

**Comment by mabdinur** [comment]
@Mergifyio backport 1.4

**Pull request merged**